sources/ippcp/pcpsms4_ctr_gfni.c

```diff
@@ -43,12 +43,12 @@
 
 #include "pcpsms4_gfni.h"
 
-static __ALIGN32 Ipp8u endiannes_swap[] = {12,13,14,15, 8,9,10,11, 4,5,6,7, 0,1,2,3,
+static __ALIGN32 Ipp8u endianness_swap[] = {12,13,14,15, 8,9,10,11, 4,5,6,7, 0,1,2,3,
```

Comment:
We have decided to postpone this change as it could affect correctness of the algorithm. Will be investigated later